website/static/js/components/socialshare.js

```diff
@@ -48,8 +48,14 @@ var ShareButtonsPopover = {
         this.popupShowing = false;
     },
     view: function(ctrl, options) {
+        var selector;
+        if (options.type === 'link') {
+            selector = 'a#sharePopoverBtn[text=Share]';
+        } else {
+            selector = 'a#sharePopoverBtn.btn.btn-default.glyphicon.glyphicon-share';
+        }
         return [
-            m('a#sharePopoverBtn.btn.btn-default.glyphicon.glyphicon-share[href=#][data-toggle=popover]', {
+            m(selector + '[href=#][data-toggle=popover]', {
```

Comment:
The facebook sharing link isn't working for me, but then neither is the one on staging.  Is there something else I need to set up?

Reply:
Yes! I was mostly copying the structure of the calls above it, but basically if there are access requests, meaning `arm` is defined, then whenever the window is resized, also call the `AccessRequestManager`'s `onWindowResize` method, which takes care of setting `collapsed()` according to parameters, much like the contributors view model does

Reply:
It's strange, the links are working for me sporadically both locally on this branch and locally on develop, but it's consistent between the 2 at least.... I wonder if it's something to do with the popup being dismissed onclick? It seems like even on develop the same issue is happening. Will investigate! In either case I didn't change any of the button/popup functionality, just allowed the popover to come up on a link or a button, so this might be an unrelated bug!

Reply:
Should we reticket the facebook sharing link?